NPB3.3-MPI/cmd/run.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
It could be great to add a command for listing the benchmarks which have been compiled before